gpustack/routes/proxy.py

```diff
@@ -0,0 +1,64 @@
+from fastapi.responses import JSONResponse
+import httpx
+import logging
+
+from fastapi import APIRouter, Request
+
+from gpustack.api.exceptions import (
+    BadRequestException,
+    ForbiddenException,
+)
+from gpustack.server.deps import SessionDep
+
+router = APIRouter()
+
+logger = logging.getLogger(__name__)
+
+
+WHITELISTED_DOMAINS = ["https://modelscope.cn", "https://www.modelscope.cn"]
```

Comment:
`ALLOWED_DOMAINS`: There's a trend to use 'allowlist' instead of 'whitelist' nowadays. And a domain usually contains no protocol.